foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/AbstractPomMigration.java

```diff
@@ -0,0 +1,43 @@
+package com.boozallen.aissemble.upgrade.migration;
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.technologybrewery.baton.BatonException;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.stream.Stream;
+
+public abstract class AbstractPomMigration extends AbstractAissembleMigration {
+    protected static final Logger logger = LoggerFactory.getLogger(AbstractPomMigration.class);
+
+    public static final String POM = "pom";
+    public static final String GUARANTEED_TAG = "<modelVersion>";
+
+    protected String indent;
+    
+    protected void detectAndSetIndent(File file) {
```

Comment:
A: Probably a candidate for Baton's `PomHelper`. Do I recognize this from one of the patch release branches? If so, I might already have a branch locally to add this to Baton.

Reply:
Yes I copied this from an old branch. Guess it never got added to dev after a patch release.